File: model/city.jl
export city, createListOfcountrylanguage

struct city
    Id::Int32
    Name::String
    CountryCode::String
    District::String
    Population::Int32
end

function createListOfCity(data)

    listOfCity = []

    for row in data
        print(row)
        print(typeof(row[1]))
        country = city(row[1], row[2], row[3], row[4], row[5])
        push!(listOfCountry, country)
    end

    return listOfCity
end

File: frontend/model/city.js
class City {
    constructor(
        id,
        countryCode,
        district,
        name, 
        population
    ){
        this.id = id,
        this.countryCode = countryCode,
        this.district = district,
        this.name = name,
        this.population = population
    }

    static createCityFromJSON(jsonData){
        const city = new City(
            jsonData.Id,
            jsonData.CountryCode,
            jsonData.District,
            jsonData.Name,
            jsonData.Population
        );

        return city;
    }
}

module.exports = City;
